cfg/std.cfg

```diff
@@ -7594,12 +7594,157 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
       <not-uninit/>
     </arg>
   </function>
+  <!-- C Threads https://en.cppreference.com/w/c/thread -->
+  <!-- int thrd_create( thrd_t *thr, thrd_start_t func, void *arg ); // since C11 -->
+  <function name="thrd_create">
+    <noreturn>false</noreturn>
+    <returnValue type="int"/>
+    <leak-ignore/>
+    <arg nr="1" direction="out">
+      <not-null/>
+    </arg>
+    <arg nr="2"/>
+    <arg nr="3"/>
+  </function>
+  <!-- thrd_t thrd_current(void); // since C11 -->
+  <function name="thrd_current">
+    <noreturn>false</noreturn>
+    <returnValue type="thrd_t"/>
+    <use-retval/>
+  </function>
+  <!-- int thrd_detach( thrd_t thr ); // since C11 -->
+  <function name="thrd_detach">
+    <noreturn>false</noreturn>
+    <returnValue type="int"/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-uninit/>
+      <not-bool/>
+    </arg>
+  </function>
+  <!-- int thrd_equal( thrd_t lhs, thrd_t rhs ); // since C11 -->
+  <function name="thrd_equal">
+    <noreturn>false</noreturn>
+    <returnValue type="int"/>
+    <use-retval/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-uninit/>
+    </arg>
+    <arg nr="2" direction="in">
+      <not-uninit/>
+      <not-bool/>
+    </arg>
+  </function>
   <!-- _Noreturn void thrd_exit( int res ); // since C11 -->
   <function name="thrd_exit">
     <noreturn>true</noreturn>
     <returnValue type="void"/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-uninit/>
+    </arg>
+  </function>
+  <!-- int thrd_join( thrd_t thr, int *res ); // since C11 -->
+  <function name="thrd_join">
+    <noreturn>false</noreturn>
+    <returnValue type="int"/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-uninit/>
+    </arg>
+    <arg nr="2" direction="out">
+      <not-bool/>
+    </arg>
+  </function>
+  <!-- int thrd_sleep( const struct timespec* duration, struct timespec* remaining ); // since C11 -->
+  <function name="thrd_sleep">
+    <noreturn>false</noreturn>
+    <returnValue type="int"/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-null/>
+      <not-uninit/>
+    </arg>
+    <arg nr="2" direction="out">
+      <not-bool/>
```

Comment:
What about `<not-uninit/> `?

Reply:
That would result in false positives. The content does not need to be initialized since it is an `out` argument.